### ISSUE TITLE
No immediately obvious way to go back to the current version docs

### ISSUE THREAD

**Issue by squarefrog** [CocoaPods/cocoadocs.org#328]
Case in point:

![screenshot 2015-04-20 08 53 23](https://cloud.githubusercontent.com/assets/343450/7225868/c14666b6-e73a-11e4-8128-4639c5df8710.png)

 Arrived at [ARAnalytics 1.2](http://cocoadocs.org/docsets/ARAnalytics/1.2/) from [a link](http://artsy.github.io/blog/2013/04/10/aranalytics/). Clicking Versions shows that the latest version is [2.7.2](http://cocoadocs.org/docsets/ARAnalytics/2.7.2/), however if you search for ARAnalytics, then the latest version is [3.0.0](http://cocoadocs.org/docsets/ARAnalytics/3.0.0/).

I see the fastest way to get to the current docs is to strip the version from the URL, but this wasn't immediately obvious.

**Comment by orta** [comment]
Agree, could definitely add a "latest" in that dropdown that does that.
